mappings/net/minecraft/item/Item.mapping

```diff
@@ -118,7 +142,14 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 		ARG 1 stack
 	METHOD method_7882 getMaxCount ()I
 	METHOD method_7884 useOnBlock (Lnet/minecraft/class_1838;)Lnet/minecraft/class_1269;
+		COMMENT Called when an item is used on a block.
+		COMMENT
+		COMMENT <p>This method is called on both the logical client and logical server, so take caution when using this method.
+		COMMENT The logical side can be checked using {@link net.minecraft.world.World#isClient() context.getWorld().isClient()}.
+		COMMENT
+		COMMENT @return an action result that specifies if using the item on a block was successful.
```

Comment:
Maybe make note about how the whole interaction process happens.

**This may be wrong!!!!** I recall that on client use on block is first checked and sent to server if possible. then check using item alone and send to server. Needa check and write in detail

Reply:
Maybe add something like below to clarify part of the interaction process.
```
<p>If the action result is successful on a logical client, then the action will sent to the logical server for processing.
```

Reply:
Hmm, does the client only send the result when it succeed? needa check that minecraft client code

Reply:
yes